### ISSUE TITLE
Issue with "white-space": The PRE element itself seems to cause two line breaks

### ISSUE THREAD

**Issue by David263** [mdn/content#6752]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/white-space

#### What information was incorrect, unhelpful, or incomplete?
It looks like only line breaks of content inside PRE elements are described, not the line breaks caused by the PRE element itself.

#### Specific section or headline?
No, nothing specific.

#### What did you expect to see?
I expected to see the line-breaking of the PRE element itself mentioned.

#### Did you test this? If so, how?
Code:
<body>
<pre>xxx</pre><pre>xxx</pre>
</body>

Output (both Firefox and Chrome):
xxx

xxx

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/css/white-space`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/white-space
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/white-space/index.html
* Last commit: https://github.com/mdn/content/commit/69f5a958188c2c631e7b125992e3048625737ff1
* Document last modified: 2021-06-29T03:43:37.000Z

</details>

**Comment by Anurella** [comment]
@David263 

The <pre> element has a display of block, that is the reason behind the spacing of the two elements in your debug example, even though in your html it is written side by side.  

Let me know if it answers the question

**Comment by David263** [comment]
Yes, it sounds right. I had forgotten the difference between different CSS 'display' types. Perhaps this is worth mentioning in this article? Presumably, one can always use css to change this behavior. And perhaps others will make the same mistake I did in thinking of PRE as being like SPAN, in spite of the fact that spans containing blocks are invalid.
